CONTRIBUTING.md

```diff
@@ -266,6 +267,11 @@ By default, tests run against Broad deployment. To run against a different deplo
 For example, consider the project that external resources are created in. The Broad deployment uses a project in Broad
 GCP org; Verily deployment uses a project in Verily GCP org.
 
+#### Troubleshooting
+
+If you see `Connecting to Docker daemon failed. Check that Docker is installed and running.`
```

Comment:
If it's just for linux, then it would be good to specify that.

Reply:
Done.

I *think* this may only affect Googlers. On linux, we install docker differently from others. But I figure this is the best place, rather than some private Google place.